### ISSUE TITLE
apt-get remove ironhide broken due to missing enablecard script.

### ISSUE THREAD

**Issue by andaag** [MrMEEE/ironhide#7]
apt-get remove produces this:

Removing ironhide ...
/var/lib/dpkg/info/ironhide.postrm: 38: /usr/local/bin/ironhide-enablecard: not found
dpkg: error processing ironhide (--remove):
 subprocess installed post-removal script returned error exit status 127

Should be an easy enough fix :)

**Comment by deemee** [comment]
I got the same issue.

**Comment by MrMEEE** [comment]
Quick fix.. don't uninstall :D....

second fix: I just committed a working script to the ppa.. should be available soon..

**Comment by MrMEEE** [comment]
Please reopen if problem persists...

**Comment by Vanuan** [comment]
Fix not available yet or doesn't work. Last version: 1.0-1~nattyubuntu13

**Comment by MrMEEE** [comment]
not available yet.. fixed in 1.0-1~nattyubuntu15

**Comment by Vanuan** [comment]
BTW, quick fix would be

```
sudo touch /usr/local/bin/ironhide-enablecard 
sudo chmod +x /usr/local/bin/ironhide-enablecard 
```
